test/e2e/instrumentation/monitoring/custom_metrics_deployments.go

```diff
@@ -54,11 +60,13 @@ var (
 			},
 		},
 	}
-	StagingDeploymentsLocation = "https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/staging/"
-	AdapterForOldResourceModel = "adapter_old_resource_model.yaml"
-	AdapterForNewResourceModel = "adapter_new_resource_model.yaml"
-	AdapterDefault             = AdapterForOldResourceModel
-	ClusterAdminBinding        = "e2e-test-cluster-admin-binding"
+	StackdriverStagingDeploymentsLocation = "https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/staging/"
+	CustomStagingDeploymentsLocation      = "https://raw.githubusercontent.com/damemi/k8s-e2e-adapter/master/deploy/manifests/"
```

Comment:
This is the adapter that should get moved to the Kubernetes org somehow (whether it lives in its own repo or under this one). Right now it's just on my personal account but this PR is effectively blocked until the adapter's home is resolved

Reply:
downloading from the external world is always error prune and will not work in locked-down environment, so we should have these file in `/test/e2e/testing-manifests/` folder use them instead.

/cc @dims @ixdy

Reply:
agreed.

also, even if we were downloading from github (which is a bad idea), these would need to be versioned, since otherwise this test can randomly fail if the master branch of these repos changes.